dotcms-integration/src/test/java/com/dotcms/rest/api/v1/content/_import/ContentImportResourceIntegrationTest.java

```diff
@@ -0,0 +1,381 @@
+package com.dotcms.rest.api.v1.content._import;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+
+import com.dotcms.JUnit4WeldRunner;
+import com.dotcms.Junit5WeldBaseTest;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.jobs.business.api.JobQueueManagerAPI;
+import com.dotcms.jobs.business.job.Job;
+import com.dotcms.jobs.business.processor.impl.ImportContentletsProcessor;
+import com.dotcms.jobs.business.util.JobUtil;
+import com.dotcms.mock.response.MockHttpResponse;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.api.v1.JobQueueManagerHelper;
+import com.dotcms.rest.exception.ValidationException;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.liferay.portal.model.User;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+import org.jboss.weld.junit5.EnableWeld;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+/**
+ * Integration test suite for content import functionality.
+ * Tests the ContentImportResource API endpoints for various scenarios.
+ */
+@ApplicationScoped
+@EnableWeld
+@RunWith(JUnit4WeldRunner.class)
+public class ContentImportResourceIntegrationTest extends Junit5WeldBaseTest {
```

Comment:
This test must be included at the end of one of our MainSuite classes. Otherwise, it won't be run on cloud